src/Language/PureScript/Sugar/Operators.hs

```diff
@@ -249,9 +250,38 @@ externsFixities ExternsFile{..} =
       , Fixity assoc prec
       , name
       )
+desugarTypeClassTypeOpsForModule
```

Comment:
Would it make more sense to call this `desugarInstanceTypeOpsForModule`? Also all the other functions in this module seem to use the full word "operators" rather than abbreviating it, so I think it would be best to stick to that here too.